cmd/apps/inletsoperator_app.go

```diff
@@ -36,10 +37,10 @@ func MakeInstallInletsOperator() *cobra.Command {
 	inletsOperator.Flags().StringP("token-file", "t", "", "Text file containing token or a service account JSON file")
 	inletsOperator.Flags().StringP("secret-key-file", "s", "", "Text file containing secret key, used for providers like ec2")
 	inletsOperator.Flags().Bool("update-repo", true, "Update the helm repo")
-
 	inletsOperator.Flags().String("pro-client-image", "", "Docker image for inlets-pro's client")
 	inletsOperator.Flags().Bool("helm3", true, "Use helm3, if set to false uses helm2")
 	inletsOperator.Flags().StringArray("set", []string{}, "Use custom flags or override existing flags \n(example --set=image=org/repo:tag)")
+	inletsOperator.Flags().String("license-file", "lf", "The text file containing license key for inlets-pro")
```

Comment:
Not sure you can use a two letter alias of `lf`, was this tested?

Reply:
@alexellis, It was tested passing value to the parameter name. I will double check while making changes. Thanks.

Reply:
Let's set it to something else, a single character. is `-f` available? or `-l`?

Reply:
Sure. Thanks.